recipe/meta.yaml

```diff
@@ -14,7 +14,7 @@ source:
 
 build:
   noarch: python
-  number: {{ build }}
+  number: 0
```

Comment:
This is a good opportunity to get rid of the excess if jinja. We definitely don't need the jinja variable on line 4, specially with the bots. (I would eliminate the checksum as a jinja var too.)

Ping @CJ-Wright

Reply:
That's fair, would you mind opening an issue at cf-graph about the canonical jinja variables and using the bot to remove excess?

Reply:
The excess jinja is a deeper issue b/c that cropped up in conda-skeleton.
I'll open an issue for the `build number` issue in cf-graph.

Reply:
Thank you!